acpreprocessing/stitching_modules/convert_to_n5/tiff_to_ngff.py

```diff
@@ -406,9 +410,9 @@ class MIPArray:
 
 
 def iterate_mip_levels_from_mimgfns(
-        mimgfns, lvl, block_size, downsample_factor,
+        mimgfns, lvl, block_size, slice_length, downsample_factor,
         downsample_method=None, lvl_to_mip_kwargs=None,
-        interleaved_channels=1, channel=0):
+        interleaved_channels=1, channel=0, deskew_kwargs={}):
```

Comment:
better to do something like `deskew_kwargs=None` here and later `deskew_kwargs = (deskew_kwargs or {})` so any changes to the dictionary aren't made to the shared default.

Reply:
Hopefully fixed in a69c6b4